ospfd/ospf_vty.c

```diff
@@ -5435,6 +5436,14 @@ static int show_ip_ospf_neighbor_id_common(struct vty *vty, struct ospf *ospf,
 						     use_json);
 	}
 
+	if (json_vrf && use_json) {
```

Comment:
In what case, json_vrf can be NULL?

Reply:
json_vrf will be NULL if user wants to query neighbor in a specific VRF or ospf instance. 
Example: If user queries ```show ip ospf vrf vrf2 neighbor 1.1.1.1```  or ```show ip ospf 2  neighbor 2.2.2.2``` then "json_vrf" will be NULL. 

"json_vrf" will be NON NULL if user queries ```show ip ospf neighbor 1.1.1.1```  or  ```show ip ospf vrf all neighbor 1.1.1.1```.